book/src/chap02.asciidoc

```diff
@@ -1,67 +1,65 @@
 [[chap02]]
-== Variables, Expressions and Statements
+== Variáveis, Expressões e Declarações
+Uma das características mais poderosas de uma linguagem de programação é a sua habilidade de manipular _variáveis_. Uma variável é um nome que refere-se a um valor.
 
-One of the most powerful features of a programming language is the ability to manipulate _variables_. A variable is a name that refers to a value.
 (((variable)))(((value)))
```

Comment:
@christianpeel We were not sure what are these. I couldn't find any reference to it on the resulting html.

Reply:
Do you mean the values in parentheses such as `(((variable)))`?  If so, these are used to create an index in the PDF version of the book.  For example, see the end of the [Spanish PDF](https://introajulia.org/latest/book.pdf).   We want to translate the values in parentheses, so assuming "variable" is translated as "variável", then we would change `(((variable)))` in to `(((variável)))`

Reply:
@joaoluisro, veja a discussão, você deve traduzir essas tags também.